CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ _None_
 ### New Features
 
 - Make fastlane log actions as collapsible groups in Buildkite. [#638]
+- Add `api_url` parameter to `create_release_backmerge_pull_request` to support GitHub Enterprise. [#641]
```

Comment:
Could be worth mentioning (probably in the same list item) that the intermediate branch that action creates is also now tracking the remote branch?